src/borg/archiver.py

```diff
@@ -4230,6 +4230,9 @@ def run(self, args):
         args.progress |= is_serve
         self._setup_implied_logging(vars(args))
         self._setup_topic_debugging(args)
+        if all((hasattr(args, x) for x in ('stats', 'dry_run'))) and all((args.stats, args.dry_run)):
+            logger.error("--stats and --dry-run shouldn't be used together")
```

Comment:
guess users trying that will then ask themselves (or us): **why?**

you could also just tell "--stats does not work with --dry-run".

Reply:
right,actually i just copied the error message from your comment.